apps/admin/backend/src/store.ts

```diff
@@ -234,8 +234,22 @@ export class Store {
   saveSystemSettings(systemSettings: SystemSettings): void {
     this.client.run('delete from system_settings');
     this.client.run(
-      'insert into system_settings (are_poll_worker_card_pins_enabled) values (?)',
-      systemSettings.arePollWorkerCardPinsEnabled ? 1 : 0 // No booleans in sqlite3
+      `
+      insert into system_settings (
+        are_poll_worker_card_pins_enabled,
+        inactive_session_time_limit_minutes,
+        num_incorrect_pin_attempts_allowed_before_card_lockout,
+        overall_session_time_limit_hours,
+        starting_card_lockout_duration_seconds
+      ) values (
+        ?, ?, ?, ?, ?
+      )
+      `,
+      systemSettings.arePollWorkerCardPinsEnabled ? 1 : 0,
+      systemSettings.inactiveSessionTimeLimitMinutes,
+      systemSettings.numIncorrectPinAttemptsAllowedBeforeCardLockout,
+      systemSettings.overallSessionTimeLimitHours,
+      systemSettings.startingCardLockoutDurationSeconds
```

Comment:
Definitely want to think some more about how to best share this query logic, but that felt out of scope for this PR